testssl.sh

```diff
@@ -2217,6 +2221,46 @@ neat_list(){
                kx="$kx "
           done
      fi
+     if [[ -n "$5" ]]; then
+          # FIXME: When $5 is true, highlight ciphers based on quality.
+          if [[ "$5" == "false" ]]; then
+               str_len=${#hexcode}
+               hexcode="$(pr_litegrey "$hexcode")"
+               for (( i=str_len; i < 7; i++ )); do
+                    hexcode+=" "
+               done
+
+               str_len=${#kx}
+               kx="$(pr_litegrey "$kx")"
```

Comment:
If it is forseeable that the line is printed in grey I think it's better collecting the output of a line in a variable and then eventually do a  ``pr_litegrey $whole_line``.

If I got this right...